catchpoint_opentsdb_bridge.py

```diff
@@ -24,8 +24,13 @@ class OpenTSDBProtocol(Protocol):
 
     @defer.inlineCallbacks
     def put(self, metric_base, D, valkey, tagkeys=()):
+        def _utf8_encode(obj):
+            if isinstance(obj, (str, unicode)):
+                return obj.encode("utf-8")
```

Comment:
I think that if obj is already a utf-8 encoded string, this will double encode it. If you're just trying to eliminate unicode errors, maybe consider something like this instead?

```
    if isinstance(obj, str):
        return str
    if isinstance(obj, unicode):
        return obj.encode('utf-8')
    return str(obj)
```